src/core/symbology/qgsgraduatedsymbolrenderer.h

```diff
@@ -267,6 +322,10 @@ class CORE_EXPORT QgsGraduatedSymbolRenderer : public QgsFeatureRenderer
         const QString &attrName,
         int classes,
         Mode mode,
+        bool useSymmetricMode,
```

Comment:
we can't add these arguments in the middle of the existing ones, because that breaks stable API. They need to be added at the end and made optional.